src/package-copy.json

```
@@ -0,0 +1,29 @@
+{
+    "name": "",
+    "version": "0.0.0",
+    "description": "",
+    "main": "",
+    "bin": {
+    },
+    "scripts": {
+      "build": "tsc",
+      "start": "node"
```

```suggestion
      "start": "node ./prod/index.js"
```

Like the `main` field above.